nav2_behavior_tree/include/nav2_behavior_tree/plugins/action/compute_path_through_poses_action.hpp

```diff
@@ -75,7 +75,7 @@ class ComputePathThroughPosesAction
   {
     return providedBasicPorts(
       {
-        BT::InputPort<std::vector<geometry_msgs::msg::PoseStamped>>(
+        BT::InputPort<geometry_msgs::msg::PoseStampedArray>(
```

Comment:
All the ports need to have their config docs updated for the change in type

Reply:
I don't really see anywhere where `std::vector<geometry_msgs::msg::PoseStamped>` was specifically mentioned so that we need to change it. Can you point me more explicitly?

Reply:
An example: https://docs.nav2.org/configuration/packages/bt-plugins/actions/ComputePathThroughPoses.html

Basically, any of the BT nodes you modified, you should be able to find the `goals` type within the BT XML config docs for that respective BT node

Reply:
thanks, done in docs PR